services/dask-sidecar/tests/unit/test_file_utils.py

```diff
@@ -375,3 +376,74 @@ async def test_pull_compressed_zip_file_from_remote(
         assert file.exists()
         assert file.name in file_names_within_zip_file
     mocked_log_publishing_cb.assert_called()
+
+
+def _compute_hash(file_path: Path) -> str:
+    with file_path.open("rb") as file_to_hash:
+        file_hash = hashlib.sha256()
+        chunk = file_to_hash.read(8192)
```

Comment:
Q: any special reason for chunks of 8192 ?

Reply:
no, that comes out of @GitHK code